### PR TITLE
REF: Show reload txs button if isCatalyst

### PR DESCRIPTION
**Pull request by marcosrdz**
Big Sur doesnt show the reload button given that it reports itself as iOS

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/95uqeggf1q3bfc3maykckap22w

[download apk](http://filestorage.bluewallet.io:1488/1605500709.apk)

**Review by ncoelho — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4wu35gk9nxyptc093gqqnrmbgw

[download apk](http://filestorage.bluewallet.io:1488/1605640621.apk)

File: package.json [hunk 1]
```diff
@@ -130,6 +130,7 @@
     "react-native-haptic-feedback": "1.11.0",
     "react-native-image-picker": "2.3.4",
     "react-native-inappbrowser-reborn": "git+https://github.com/BlueWallet/react-native-inappbrowser.git#fa2d8e1763e46dd12a7e53081e97a0f908049103",
+    "react-native-is-catalyst": "git+https://github.com/BlueWallet/react-native-is-catalyst.git#v1.0.0",
```
Comment (Overtorment) [inline package.json:133]: i LOLd when I saw how much code is in this dep:  https://github.com/BlueWallet/react-native-is-catalyst/blob/master/ios/IsCatalyst.m

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**